### PR TITLE
PS-8090 Update the ZenFS doc for backup and restore settings (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
M source/myrocks/zenfs.rst

### CODE REVIEW

File: doc/source/myrocks/zenfs.rst [hunk 1]
```diff
@@ -243,23 +243,97 @@ The following command backs up everything from the root of the ZenFS drive:
 
 .. sourcecode:: bash
 
-    zenfs backup --zbd=${NULLB} --path="/home/user/bkp" --backup_path=./
+    $ zenfs backup --zbd=${NULLB} --path="/home/user/bkp" --backup_path=./
```
Comment (percona-ysorokin) [inline doc/source/myrocks/zenfs.rst]: Let's normalize backup_path / restore_path in examples - `/` is a more straightforward value meaning "root"
```suggestion
    $ zenfs backup --zbd=${NULLB} --path="/home/user/bkp" --backup_path=/
```

File: doc/source/myrocks/zenfs.rst [hunk 2]
```diff
@@ -243,23 +243,97 @@ The following command backs up everything from the root of the ZenFS drive:
...
+     * - ./<directory>/
+       - A single period, a forward slash, the directory name, and an ending forward slash
+
 
 Use the following command to restore a backup into the root of the ZenFS drive:
 
 .. sourcecode:: bash
 
-    zenfs restore --zbd=${NULLB} --path="/home/user/bkp/" --restore_path=.
+    $ zenfs restore --zbd=${NULLB} --path="/home/user/bkp/" --restore_path=.
```
Comment (percona-ysorokin) [inline doc/source/myrocks/zenfs.rst]: Likewise
```suggestion
    $ zenfs restore --zbd=${NULLB} --path="/home/user/bkp/" --restore_path=/
```

**Review by percona-ysorokin — APPROVED** [review]
LGTM with 2 minor comments

File: doc/source/myrocks/zenfs.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/myrocks/zenfs.rst]: Done.

File: doc/source/myrocks/zenfs.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/myrocks/zenfs.rst]: Done.

**Pull request merged**